sheetScript.gs

```diff
@@ -25,12 +34,10 @@ function doGet(e) {
 }
 
 function findNextRow(data) { // Finds the row with the earliest last updated time
-  var reservedTimestampColumn = 1;
-  var reservedTableRows = 3; // Start of the range of the PlaylistID+ChannelID data
   var minTimestamp = data.slice(reservedTableRows).reduce(
     function (min, row, index) {
-      if (row[reservedTimestampColumn].length != 0 && row[reservedTimestampColumn] < min[1]) {
-        return [index, row[reservedTimestampColumn]]
+      if (row[reservedColumnPlaylist].length != 0 && row[reservedColumnPlaylist] < min[1]) {
```

Comment:
Wouldn't `reservedColumnTimestamp - 1` be more readable?

Reply:
That assumes you'll never change the order of the columns again. Considering that the code contained multiple mistakes with constant offsets (reservedTimestampColumn +1 +2 +3)  I recommend making these explicit at the beginning of the script.

Reply:
Do whatever you think makes sense, and allows future developers to understand what's going on.